helix-tui/src/backend/crossterm.rs

```diff
@@ -31,11 +31,22 @@ fn vte_version() -> Option<usize> {
     std::env::var("VTE_VERSION").ok()?.parse().ok()
 }
 
+#[derive(Clone, Debug)]
+struct ResetCursorCommand(String);
```

Comment:
The newtype wrapper for the string here seems unnecessary to me. It's fine just to default with `unwrap_or`/`unwrap_or_else` rather than `unwrap_or_default`

Reply:
I added this since we also want the default `"\x1B[0 q"` to be used when `terminfo` isn't available, for example on Windows, or else it just defaults to an empty string. Or should I just explicitly make the struct in the other case instead of keeping the `Capabilities::default()`?

Reply:
Ah I see. Let's define `impl Default for Capabilities` instead so Windows can keep calling `Capabilities::default`